tests/Behat/Transform/TaxonContext.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Transform;
```

Comment:
This transformer is already in the plugin, but in the `Tests\Sylius\PriceHistoryPlugin\Behat\Context\Transform` namespace, which is the correct one. Merge this one with this already existing.